src/containers/tutor-home-page/language-step/LanguageStep.jsx

```diff
@@ -26,7 +28,9 @@ const LanguageStep = ({ btnsBox }) => {
   const handleClear = () => {
     setSelectedLanguage('')
   }
-
+  useEffect(() => {
+    handleStepData(stepLabel, selectedLanguage)
+  }, [selectedLanguage])
```

Comment:
Please pay attention to the linter warning here.

Reply:
I noticed it and checked, and everything is okay with these dependencies)

Reply:
No, it is not okay, `handleStepData` and `stepLabel` are external function for this effect, so they must be added as dependencies.